source/math/FloatVectorList.ooc

```diff
@@ -382,4 +383,68 @@ FloatVectorList: class extends VectorList<Float> {
 			result add(this[i] clamp(floor, ceiling))
 		result
 	}
+	findMaxIndex: func -> Int {
```

Comment:
`findMaximumIndex`. Although it's easy to assume this actually returns `this count - 1`... So `findIndexOfMaximum` would be more specific.